cmd/manager/main.go

```diff
@@ -292,6 +293,16 @@ func Command() *cobra.Command {
 		"0.0.0.0",
 		fmt.Sprintf("The host to which the operator should bind to serve metrics in the Prometheus format. Will be combined with %s.", operator.MetricsPortFlag),
 	)
+	cmd.Flags().Bool(
+		operator.MetricsSecureFlag,
```

Comment:
Should we also update [operator-config.asciidoc](https://github.com/elastic/cloud-on-k8s/blob/ec87ca75f8e613a78113a6a19f5e2817b73057b5/docs/operating-eck/operator-config.asciidoc?plain=1#L39)?